scripts/get_maintainer.py

```diff
@@ -31,13 +31,7 @@
 import subprocess
 import sys
 
-from yaml import load, YAMLError
-try:
-    # Use the speedier C LibYAML parser if available
-    from yaml import CLoader as Loader
-except ImportError:
-    from yaml import Loader
```

Comment:
Why did you remove this part?

Reply:
`safe_load` does not support specifying a loader afaik, the `load()` was the only use case of those imports

Reply:
Actually replaced it with SafeLoader.